src/index.html

```diff
@@ -0,0 +1,194 @@
+                required
+                minlength="2"
+                maxlength="40"
+              />
+              <span id="profile-name-input-error" class="modal__error"></span>
+            </label>
+            <label for="profile-description-input" class="modal__label">
+              Description
+              <input
+                id="profile-description-input"
+                class="modal__input"
+                placeholder="Describe yourself"
+                required
+                minlength="2"
+                maxlength="200"
+              />
+              <span
+                id="profile-description-input-error"
+                class="modal__error"
+              ></span>
+            </label>
+            <button type="submit" class="modal__submit-btn">Save</button>
+          </form>
+        </div>
+      </div>
+    </div>
+
+    <div class="modal" id="add-card-modal">
+      <div class="modal__container">
+        <button type="button" class="modal__close-btn"></button>
+        <h2 class="modal__title">New post</h2>
+        <form name="edit-profile" class="add-card-form modal__form">
+          <label for="add-card-link-input" class="modal__label">
+            Image link
+            <input
+              id="add-card-link-input"
+              name="link"
+              class="modal__input"
+              placeholder="Paste a link to the picture"
+              type="url"
+              required
+            />
+            <span id="add-card-link-input-error" class="modal__error"></span>
+          </label>
+          <label for="add-card-name-input" class="modal__label">
+            Caption
+            <input
+              id="add-card-name-input"
+              name="name"
+              class="modal__input"
+              placeholder="Type your caption"
+              type="text"
+              required
+              minlength="2"
+              maxlength="30"
+            />
+            <span id="add-card-name-input-error" class="modal__error"></span>
+          </label>
+          <button type="submit" class="modal__submit-btn">Save</button>
+        </form>
+      </div>
+    </div>
+
+    <div class="modal modal" id="preview">
+      <div class="modal__container modal__container_type_preview">
+        <button
+          aria-label="Close modal"
+          class="modal__close-btn modal__close-btn_type_preview"
+        ></button>
+        <img class="modal__image" src="" alt="" />
+        <p class="modal__caption"></p>
+      </div>
+    </div>
+
+    <div class="modal" id="avatar-modal">
+      <div class="modal__container">
+        <button type="button" class="modal__close-btn"></button>
+        <h2 class="modal__title">Edit profile avatar</h2>
+        <form name="edit-avatar-profile" id="modal_form" class="modal__form" novalidate>
+          <label for="profile-avatar-input" class="modal__label">
+           Profile Picture
+            <input
+              id="profile-avatar-input"
```

Comment:
* Please, add `type="url"` for the `avatar` input, to validate it as a `link`

Reply:
My remark hasn't been corrected https://github.com/Jae277/se_project_spots/pull/1#discussion_r2097622897